client/src/legacy/TinyMCE_sslink-anchor.js

```diff
@@ -157,6 +158,9 @@ jQuery.entwine('ss', ($) => {
 });
 
 // Adds the plugin class to the list of available TinyMCE plugins
-tinymce.PluginManager.add(commandName, (editor) => plugin.init(editor));
+tinymce.PluginManager.add(commandName, (editor) => { 
+  editorIdentifier = editor.getParam('editorIdentifier');
+  plugin.init(editor)
+});
```

Comment:
This happens too late in the process. By the time we do this, we've _already_ passed in `undefined` to `TinyMCEActionRegistrar.addAction` which means it doesn't know what editor it's creating the action for.

You can verify that by adding `console.log({configId});` into the `addAction` function.

I don't think there's a way to know _which_ editor we want at this stage. It also doesn't seem like there's any downside to _not_ specifying an editor.... so maybe we can just remove `configId` from the `TinyMCEActionRegistrar` altogether.

The only other alternative I can think of is to add the action inside the plugin's `init()` function (right before `addCommand()`). Try this before trying to remove it altogether.

Reply:
This comment applies to everywhere else you've done this as well.

Reply:
> The only other alternative I can think of is to add the action inside the plugin's `init()` function (right before `addCommand()`). Try this before trying to remove it altogether.

This doesn't work if user has more than one Content Editor component on the page.

Reply:
> I don't think there's a way to know _which_ editor we want at this stage. It also doesn't seem like there's any downside to _not_ specifying an editor.... so maybe we can just remove `configId` from the `TinyMCEActionRegistrar` altogether.

We cannot get rid of this argument, since if user has it's own WYSIWYG editor configuration and has special name, I think TinyMCEActionRegistrar will not be able to recognise where it should append action.

Reply:
> We cannot get rid of this argument, since if user has it's own WYSIWYG editor configuration and has special name, I think TinyMCEActionRegistrar will not be able to recognise where it should append action.

~~My assumption is that this doesn't actually matter. I tried this with multiple tinymce editors on a page and couldn't see any problems resulting from this. Note that the actions have an anonymous function in `onAction` which takes the editor as an argument - that editor will be the correct instance the plugin is being activated from. So at the time the action is actually _performed_ it has the correct instance of the editor. I don't think it needs it outside of that.~~
Ignore that... I think it does matter if, for example, one of the editors should have less options in its action dropdown field (maybe one can't have links to a file for example).

I see also that my init idea doesn't work because `sslink` is then loaded before the other plugins have initialised and therefore it doesn't have their actions loaded in time. The same thing happens if we move that logic down into the `tinymce.PluginManager.add()` anonymous function...